crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -2809,8 +2817,16 @@ impl<Codec: RawEntryCodec> StateMachine<Codec> {
                     .await?;
                 }
             }
-            InvocationStatus::Killed(_) => {
-                todo!()
+            InvocationStatus::Killed(metadata) => {
+                self.send_response_to_sinks(
+                    ctx,
+                    vec![attach_invocation_request.response_sink],
+                    KILLED_INVOCATION_ERROR,
+                    Some(invocation_id),
+                    None,
+                    Some(&metadata.invocation_target),
+                )
+                .await?;
```

Comment:
Could we just append a response sink? Then we have fewer cases how we generate and send responses.

Reply:
This would need to write again the invocation status I think, I think it's ok to keep it like this as long as these states are very clear...